experimental/packages/opentelemetry-instrumentation-grpc/src/enums/AttributeNames.ts

```diff
@@ -17,9 +17,13 @@
 /**
  * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md
  */
-export enum AttributeNames {
-  GRPC_KIND = 'grpc.kind', // SERVER or CLIENT
-  GRPC_METHOD = 'grpc.method',
-  GRPC_ERROR_NAME = 'grpc.error_name',
-  GRPC_ERROR_MESSAGE = 'grpc.error_message',
+
+interface AttributesType {
+  GRPC_ERROR_NAME: string;
+  GRPC_ERROR_MESSAGE: string;
 }
+
+export const AttributeNames: Readonly<AttributesType> = {
+  GRPC_ERROR_NAME: 'grpc.error_name',
+  GRPC_ERROR_MESSAGE: 'grpc.error_message',
```

Comment:
Why not using the semantic conventions package?

Reply:
Because PR should resolve at least one issue :)
This was before, and pr resolves different issue

Reply:
I don't see a linked issue. Since the PR is about fixing the semantic conventions, I think it is also appropriate to update it to use the semconv package. The only reason this doesn't is because the package didn't exist when we wrote this instrumentation

Reply:
Up to you. It can always be done separately

Reply:
The main point is that in the beginning it was not exactly about semantics, but about attributes consistency and the same behavior for already existing otel agents

I made some easy fixes about semantics, because of requests, but I don't want to resolve all conventions that were broken before this PR. I just want to resolve the exact issue: missed attributes in grpc instrumentations :)